### PR TITLE
added a simple power driver for YEPKIT YKUSH switchable USB hubs - th…

### PR DESCRIPTION
**Pull request by tgbloks**
…is currently "highjacks" the NetworkPowerPort resource by using the host entry as serial for the USB hub. 
works for me as it is but i might be willing to tweak some stuff if you are interested in getting it into your repo ;-)

Signed-off-by: Tobi Gschwendtner <tg@bloks.de>

### CODE REVIEW

**Comment by jluebbe** [comment]
Thanks for the PR! I have some comments, though...

This should have it's own resource and driver instead of reusing NetworkPowerPort. For an example of a standalone resource look at resource/onewireport.py. You'd use serial and index as parameters.
For the driver, copy the NetworkPowerDriver and call directly to pykush. Do the import in the ``__attrs_post_init__`` method to avoid the module load overhead if it's not used. Also please map to the ``YKUSH_PORT_STATE_*`` constants defined by the pykush library.

I'd suggest YKUSHPowerPort for the resource and YKUSHPowerDriver for the driver.

**Comment by tgbloks** [comment]
no worries - i totally agree with your proposal (as it sounds just like the "nice" long term solution in my head ;-)
what you see in this PR is just the minimal effort i needed to invest to test labgrid with my available HW
(i very much like what i see so far - thx!) and since i could not find a better place to discuss this (any plans for a mailing list / irc channel?) i started here.
i will update the PR once i had time to implement and test the changes...

File: labgrid/resource/ykushpowerport.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@attr.s(cmp=False)
+class YKUSHPowerPort(Resource):
+    """This resource describes a YEPKIT YKUSH switchable USB hub.
+
+    Args:
+        serial (str): serial of the YKUSH device 
+        index (str): port index"""
+    serial = attr.ib(validator=attr.validators.instance_of(str))
+    index = attr.ib(validator=attr.validators.instance_of(str),
+                    convert=lambda x: str(int(x)))
```
Comment (jluebbe) [inline labgrid/resource/ykushpowerport.py]: If you use ``convert=lambda x: int(x)``, you should be able to remove the int() conversions in the on/off functions.

**Comment by tgbloks** [comment]
should be ok now...

**Comment by jluebbe** [comment]
Thanks for your contribution!

**Pull request merged**